src/components/index.ts

```diff
@@ -9,11 +9,26 @@ import LinearProgressBar, { LinearProgressBarProps } from './LinearProgressBar/L
 import Switch, { SwitchProps } from './Switch/Switch'
 import TextInput, { TextInputProps } from './TextInput/TextInput'
 import Toggle, { ToggleProps } from './Toggle/Toggle'
-
+import CircularLoader from './CircularLoader/CircularLoader'
+import {
+	ExpandMore,
```

Comment:
Each Icon should have "Icon" as a suffix in the name. Here it should be ExapndMoreIcon, so that it can distinctive from the components.